docs/rules/static-property-placement.md

```diff
@@ -126,23 +125,23 @@ MyComponent.propTypes = { /*...*/ };
 
 ## Rule Options
 
-```
+```plaintext
 ...
 "react/static-property-placement": [<enabled>]  // `static public field` enabled
 ...
 ```
 
 or alternatively:
 
-```
+```plaintext
```

Comment:
these are keys in an eslint config, so let's make them
```suggestion
```json
```

Reply:
Updated.